packages/rocketchat-user-data-download/server/cronProcessDownloads.js

```diff
@@ -291,7 +291,7 @@ const sendEmail = function(userId) {
 	}
 	const userData = RocketChat.models.Users.findOneById(userId);
 
-	if (!userData || userData.emails || userData.emails[0] || userData.emails[0].address) {
+	if (!userData || !userData.emails || !userData.emails[0] || !userData.emails[0].address) {
```

Comment:
I think we can improve the logic here, by looping into the emails and getting the first verified maybe? And if there's no email verified, get the first one. 🤔

Reply:
I agree this could be improved.. but not on this PR.. this is just merging the fix for the release.